### PR TITLE
improve package summary and description

### PR DESCRIPTION
**Pull request by remicollet**
To enlight open sources alternatives to Redis

Copy/paste from README.md

### CODE REVIEW

**Comment by remicollet** [comment]
This keep "redis" as the first choice

Alternative proposal

```
   This extension provides an API for communicating with key-value store.
   It supports Redis and its open source alternatives KeyDB and Valkey.

```

File: package.xml [hunk 1]
```diff
@@ -5,9 +5,10 @@ http://pear.php.net/dtd/package-2.0
 http://pear.php.net/dtd/package-2.0.xsd">
  <name>redis</name>
  <channel>pecl.php.net</channel>
- <summary>PHP extension for interfacing with Redis</summary>
+ <summary>PHP extension for interfacing with key-value store</summary>
```
Comment (michael-grunder) [inline package.xml]:
```suggestion
 <summary>PHP extension for interfacing with key-value stores</summary>
```

File: package.xml [hunk 2]
```diff
@@ -5,9 +5,10 @@ http://pear.php.net/dtd/package-2.0
...
  <description>
-   This extension provides an API for communicating with Redis servers.
+   This extension provides an API for communicating with Redis key-value store.
+   It also supports KeyDB and Valkey, which are open source alternatives to Redis.
```
Comment (michael-grunder) [inline package.xml]:
```suggestion
   This extension provides an API for communicating with RESP-based key-value stores, such as Redis, Valkey, and KeyDB.
```

**Comment by michael-grunder** [comment]
Would you like this included in 6.1.0 GA?

File: package.xml (hunk 1 above)
Comment (remicollet) [reply package.xml]: done

File: package.xml (hunk 2 above)
Comment (remicollet) [reply package.xml]: done

**Comment by remicollet** [comment]
> Would you like this included in 6.1.0 GA?

Yes please

Context: having a description saying "_for redis_" in Linux distributions without redis may sounds strange ;)

**Review by michael-grunder — APPROVED** [review]

**Pull request merged**